### PR TITLE
Try hardcoded path for NuGetFallbackFolder

### PR DESCRIPTION
**Pull request by rfrerebe**
in addition to BinFolder (working only on windows)

#3662

### CODE REVIEW

File: src/Paket.Core/Dependencies/NuGetCache.fs [hunk 1]
```diff
@@ -331,13 +331,28 @@ let GetTargetUserNupkg (packageName:PackageName) (version:SemVerInfo) =
...
     DirectoryInfo(Path.Combine(Constants.UserNuGetPackagesFolder,".tools",packageName.CompareString,version.Normalize())).FullName
 
-let TryGetFallbackFolder () =
+let TryGetFallbackFolderFromHardCodedPath () =
+    let fallbackDir =
+        match isUnix with
+        | true ->
+            [|"/usr/share/dotnet/sdk/NuGetFallbackFolder" |]
+        | false ->
+            [| @"C:\Program Files\dotnet\sdk\NuGetFallbackFolder";
```
Comment (forki) [inline src/Paket.Core/Dependencies/NuGetCache.fs]: this should read the beginning of the path from env settings

**Comment by forki** [comment]
thanks. Can you please try 5.219.4 - I tried to make it slightly more correct with using env variables

**Comment by rfrerebe** [comment]
Hello,
I can confirm that issue is fixed using 5.219.4
Thanks for the merge 🚀 

About env variables, I can submit a new PR.
I tried to use NuGetCachePath following https://fsprojects.github.io/Paket/caches.html#NuGet-machine-cache
but it did not work.
Is it what your are talking about ?

**Comment by rfrerebe** [comment]
Thanks I have seen your commit.
I learned somethign with the use of lazy

**Pull request merged**